lib/rack/handler/puma.rb

```diff
@@ -93,16 +93,16 @@ def self.set_host_port_to_config(host, port, config)
           config.bind "unix://#{host}"
         elsif host && host =~ /^ssl:\/\//
           uri = URI.parse(host)
-          uri.port ||= port || ::Puma::Configuration::DefaultTCPPort
+          uri.port ||= port || ::Puma::Configuration::DEFAULTS[:tcp_port]
```

Comment:
Question 🙋🏻‍♀️  

Should this read from a method `Puma.options` (that returns `#puma_default_options`), so it reads values from user configurations, otherwise it always defaults to puma’s default (`9292`)?

Same questions for other `::Puma::Configuration::DEFAULTS` in this Pull Request 🙇🏻‍♂️

Reply:
I see now we are only keeping the same behavior as before 👍🏼